src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.ps1

```diff
@@ -151,6 +151,67 @@ function Test-VirtualNetworkGatewayCRUD
 .SYNOPSIS
 Virtual network gateway tests
 #>
+function Test-VirtualNetworkGatewayGenerateVpnProfile
+{
+param 
+    ( 
+        $basedir = ".\" 
+    )
+
+    # Setup
+    $rgname = Get-ResourceName
+    $rname = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+    $vnetName = Get-ResourceName
+    $publicIpName = Get-ResourceName
+    $vnetGatewayConfigName = Get-ResourceName
+    $rglocation = "WEST US 2"
```

Comment:
Do not hard code locations.  This is a maintenance issue.  There is no reason you can't use the provider location below for resource groups as well.